test/test-applications/integrations/Samples.OracleMDA.Core/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Data;
+using System.Threading;
+using System.Threading.Tasks;
+using Oracle.ManagedDataAccess.Client;
+using Samples.DatabaseHelper;
+
+namespace Samples.OracleMDA.Core
+{
+    internal static class Program
+    {
+        private static async Task Main()
+        {
+            var tableId = Guid.NewGuid().ToString("N").Substring(0, 10);
+            var commandFactory = new OracleDbCommandFactory($@"oracletest{tableId}");
+            var commandExecutor = new OracleCommandExecutor();
+            var cts = new CancellationTokenSource();
+
+            using (var connection = OpenConnection())
+            {
+                await RelationalDatabaseTestHarness.RunAllAsync<OracleCommand>(connection, commandFactory, commandExecutor, cts.Token);
```

Comment:
Note that you can use `CancellationTokenSource.None` when you don't actually plan on cancelling the token

Reply:
done in 85deb15